addons/mail/static/src/models/message_view.js

```diff
@@ -140,6 +140,17 @@ registerModel({
                 messagingAsClickedMessageView: clear(),
             });
         },
+        /**
+         * This listens to the right click event, and used to redirect the event
+         * as a click on the popover.
+         *
+         * @param {Event} ev
+         */
+         async onContextMenu(ev) {
+            ev.stopPropagation();
+            return this.messageContextMenu.update({ messageContextPopoverView: {} });
```

Comment:
Should be `this.update({ messageContextPopoverView: {}  })`, and you must define field `MessageView/messageContextPopoverView` as follow:

```js
messageContextPopoverView: one('PopoverView', {
   inverse: 'messageViewOwnerAsContextMenu',
}),
```